techmatters/hrm
# CHI-2075: Rework reindex admin endpoint to stream out results as they are available

## Description

* Apply 2 new indexes to the resources schema to optimise the initial pull query
* Update the endpoint to stream out results of posting an index job as they happen - this will give real time feedback and prevent the query timing out
* Similarly update the script to push those CSV lines to stdout as they are received
* Corrected naming mistake in previous script. The entire script is idempotent so it should pose no issues if it gets run a second time on any database

### Checklist
- N/A New tests added

### Related Issues
CHI-2075

### Verification steps

Use the reindex script for a large job with the `--verbose` flag set, you should see results coming in as they are sent, and the query will not time out
